### ISSUE TITLE
accessibility - Fix 'Back to Home' issue raised during PRBuilds by Pa11y 

### ISSUE THREAD

**Issue by katebee** [guardian/frontend#18012]
**Plea: This would be a great place for someone to start with their first PR, so ideally I would love it to be picked up by someone relatively new to open source contributions**

First time contributors are very welcome! Please message @katebee if you have any questions 😁

# ISSUE

[Pa11y](http://pa11y.org/) is an automated testing tool for accessibility issues.

We run pa11y as part of our PRBuild process whenever someone opens a PR. Pa11y currently reports an error with the header:

```
ERROR: This element has insufficient contrast at this conformance level. 
Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1:1. 
Recommendation: change text colour to #feffff.
```

```
Context: <span class="u-h">The Guardian - Back to home</span>
```

The affected element is actually hidden on the website (our css class 'u-h' means it will be hidden). So the colour does not matter, but it would be helpful to have this fixed so we can work towards having zero errors reported by pa11y.

In this case it would help to apply the proper font colour to the element.

## Steps to Reproduce

Run pa11y against frontend

## Actual Results (include screenshots)

A pa11y report is generated and included in the PRBuilds report that is added as a comment to the end of every pull request, for example:

<img width="784" alt="picture 43" src="https://user-images.githubusercontent.com/8607683/31620305-cf6751f2-b28e-11e7-9165-2a179af8aa07.png">

Example pa11y report here: https://s3-eu-west-1.amazonaws.com/prbuilds/PR-17516/a11y-report.txt

## Expected Results (include screenshots)

One less error in the pa11y report
The number next to A11y validation should decrease

**Comment by katebee** [comment]
https://github.com/guardian/frontend/pull/18011

Ah well! 😅
